piet/src/gradient.rs

```diff
@@ -221,6 +264,43 @@ impl LinearGradient {
     }
 }
 
+impl RadialGradient {
+    /// Creates a simple `RadialGradient`. This gradient has the same `origin`
```

Comment:
Would it make sense to have origin and center default to center? I think that's the most common use case, and makes sense as a default, but this is also ok.

Reply:
Good idea, I think that's worth the simplification.